tests/e2e/containers/containers.go

```diff
@@ -62,7 +62,7 @@ func (m *Manager) ExecTxCmd(t *testing.T, chainId string, containerName string,
 // namely adding flags `--chain-id={chain-id} -b=block --yes --keyring-backend=test "--log_format=json"`,
 // and searching for `successStr`
 func (m *Manager) ExecTxCmdWithSuccessString(t *testing.T, chainId string, containerName string, command []string, successStr string) (bytes.Buffer, bytes.Buffer, error) {
-	allTxArgs := []string{fmt.Sprintf("--chain-id=%s", chainId), "-b=block", "--yes", "--keyring-backend=test", "--log_format=json"}
+	allTxArgs := []string{fmt.Sprintf("--chain-id=%s", chainId), "-b=block", "--yes", "--keyring-backend=test", "--log_format=json", "--fees=875uosmo"}
```

Comment:
This seems to be failing on E2E when executing https://github.com/osmosis-labs/osmosis/blob/nicolas/packet-forward-middleware/tests/e2e/configurer/upgrade.go#L140. 

It seems to me like this the user just doesn't have the funds to pay for fees.